quarkus-integration/quarkus/deployment/src/main/java/ai/timefold/solver/quarkus/deployment/TimefoldProcessor.java

```diff
@@ -573,6 +577,43 @@ private SolverConfig loadSolverConfig(IndexView indexView,
         return solverConfig;
     }
 
+    @BuildStep
+    void buildConstraintMetaModel(SolverConfigBuildItem solverConfigBuildItem,
```

Comment:
This is new code? I don't understand. Didn't we already have support for this, and therefore the code should have been either reused, or moved from elsewhere?

Reply:
Moved the shared code into a util class.